ark/utils/data_utils.py

```diff
@@ -120,49 +132,63 @@ def generate_cell_cluster_mask(fovs, base_dir, seg_dir, cell_consensus_name,
 
     if not os.path.exists(os.path.join(base_dir, cell_consensus_name)):
         raise FileNotFoundError(
-            "consensus_dir %s does not exist in base_dir %s" % (cell_consensus_name, base_dir)
+            "consensus_dir %s does not exist in base_dir %s"
+            % (cell_consensus_name, base_dir)
         )
 
     # verify the cluster_col provided is valid
     verify_in_list(
         provided_cluster_col=cell_cluster_col,
-        valid_cluster_cols=['cell_som_cluster', 'cell_meta_cluster']
+        valid_cluster_cols=["cell_som_cluster", "cell_meta_cluster"],
     )
 
     # load the consensus data in
-    cell_consensus_data = feather.read_dataframe(os.path.join(base_dir, cell_consensus_name))
+    cell_consensus_data = feather.read_dataframe(
+        os.path.join(base_dir, cell_consensus_name)
+    )
 
     # ensure the cluster col will be displayed as an integer and not a float
-    cell_consensus_data[cell_cluster_col] = cell_consensus_data[cell_cluster_col].astype(int)
+    cell_consensus_data[cell_cluster_col] = cell_consensus_data[
+        cell_cluster_col
+    ].astype(int)
 
     # verify all the fovs are valid
-    verify_in_list(
-        provided_fovs=fovs,
-        consensus_fovs=cell_consensus_data['fov']
-    )
+    verify_in_list(provided_fovs=fovs, consensus_fovs=cell_consensus_data["fov"])
```

Comment:
I don't see the pattern of the style changes being made. Most of them are expanding from one line to multiple, but this is going the opposite direction?